Makefile

```diff
@@ -1,95 +1,105 @@
+PHONY: init-%
+init-local: # @HELP Initialize terraform state for your local setup. This does not create any resources
+init-staging: # @HELP Initialize terraform state for staging. This does not create any resources
+init-production: # @HELP Initialize terraform state for production. This does not create any resources
+init-%: ENV=$*
+init-%:
+	cd ./tf/env/$(ENV) && terraform init
+
+.PHONY: diff-%
+diff-local: # @HELP Diff the repository against the state of your local cluster
+diff-staging: # @HELP Diff the repository against the state of the staging cluster
+diff-production: # @HELP Diff the repository against the state of the production cluster
+diff-%: ENV=$*
+diff-%:
+	cd ./tf/env/$(ENV) && terraform plan
+	cd ./k8s/helmfile && helmfile --environment $(ENV) diff --context 5
+
+.PHONY: apply-%
+apply-local: # @HELP Apply changes in the repository to your local cluster
+apply-staging: # @HELP Apply changes in the repository to the staging cluster
+apply-production: # @HELP Apply changes in the repository to the production cluster
+apply-%: ENV=$*
+apply-%:
+	cd ./tf/env/$(ENV) && terraform apply
+	cd ./k8s/helmfile && helmfile --environment $(ENV) --interactive apply --context 5
+
+diff: # @HELP Run diff for both staging and production
 diff: diff-staging diff-production
+apply: # @HELP Run apply for both staging and production
 apply: apply-staging apply-production
 
-.PHONY: diff-staging apply-staging
-diff-staging:
-	cd ./tf/env/staging && terraform plan
-	cd ./k8s/helmfile && helmfile --environment staging diff --context 5
-# Note: the staging command here actually terraform applies all of staging
-apply-staging:
-	cd ./tf/env/staging && terraform apply
-	cd ./k8s/helmfile && helmfile --environment staging --interactive apply --context 5
+.PHONY: test
+test: # @HELP Run yamllint tests against the repsoitory
+test:
+	yamllint --no-warnings .
 
-.PHONY: diff-production apply-production
-diff-production:
-	cd ./tf/env/production && terraform plan
-	cd ./k8s/helmfile && helmfile --environment production diff --context 5
-# Note: the production command here actually terraform applies all of production
-apply-production:
-	cd ./tf/env/production && terraform apply
-	cd ./k8s/helmfile && helmfile --environment production --interactive apply --context 5
+skaffold-mediawiki: skaffold-mediawiki-137-fp
+skaffold-mediawiki: # @HELP Run the local mediawiki module using skaffold
+skaffold-ui: # @HELP Run the local ui module using skaffold
+skaffold-api: # @HELP Run the api module using skaffold
+skaffold-queryservice: # @HELP  Run the local queryservice module using skaffold
+skaffold-queryservice-ui: # @HELP Run the local queryservice-ui module using skaffold
+skaffold-queryservice-updater: # @HELP Run the local queryservice-updater module using skaffold
+skaffold-queryservice-gateway: # @HELP Run the local queryservice-gateway module using skaffold
+.PHONY: skaffold-%
+skaffold-%: MODULE=$*
+skaffold-%:
+	cd ./skaffold && skaffold run -m $(MODULE)
```

Comment:
Is there a reason we don't pass the `kube-context` when running a single module?